rele/middleware.py

```diff
@@ -43,7 +57,12 @@ def pre_publish(self, topic, data, attrs):
         """
 
     def post_publish(self, topic):
-        """Called after Publisher sends message.
+        """DEPRECATED: Called after Publisher sends message.
+        :param topic:
+        """
+
+    def post_publish_success(self, topic):
+        """DEPRECATED: Called after Publisher sends message.
```

Comment:
I imagine this `DEPRECATED` was added by mistake, right? If I remember correctly, it's meant to go under `post_publish_success` only

Reply:
Yeah, thanks!